open_spiel/python/algorithms/expected_game_score_test.py

```diff
@@ -35,6 +36,16 @@ def test_expected_game_score_uniform_random_kuhn_poker(self):
         game.new_initial_state(), [uniform_policy] * 2)
     self.assertTrue(np.allclose(uniform_policy_values, [1 / 8, -1 / 8]))
 
+  def test_expected_game_score_uniform_random_iterated_prisoner_dilemma(self):
+    game = pyspiel.load_game(
+      "python_iterated_prisoners_dilemma(max_game_length=6)")
+    uniform_policy = policy.UniformRandomPolicy(game)
+    uniform_policy_values = expected_game_score.policy_value(
+        game.new_initial_state(), uniform_policy)
+    print(uniform_policy_values)
+    self.assertTrue(
+      np.allclose(uniform_policy_values, [19.43373108, 19.43373108]))
```

Comment:
Can you explain these values?
I would have expected 4 per round * expected number of rounds, i.e. around 17.6
(perhaps the game is lasting longer than the specific maximum number of rounds?)

Reply:
This is correct, the value should be around 17.6385498. I have changed the game length implementation (current time step at initialization is now 1).